dace/sdfg/analysis/writeset_underapproximation.py

```diff
@@ -689,21 +677,34 @@ def _merge_subsets(subset_a: subsets.Subset, subset_b: subsets.Subset) -> subset
         return subset_b
 
 
+class UnderapproximateWritesDictT(TypedDict):
```

Comment:
```suggestion
class UnderapproximateWritesDict(TypedDict):
```
Python standards would not use T at the end unless it's a TypeVar (and this is not one)